runhouse/servers/cluster_servlet.py

```diff
@@ -214,6 +218,26 @@ async def aclear_all_references_to_env_servlet_name(self, env_servlet_name: str)
     ##############################################
     # Periodic Cluster Checks APIs
     ##############################################
+
+    def _observability_disabled(self) -> bool:
+        """If disable_observability value was changed on the cluster level, it was changed on the local
```

Comment:
is there any reason to even look at the value in the config.yaml on the cluster? since when we restart the runhouse server we'll be saving whatever value was passed into the `restart` command directly into the cluster servlet?